src/components/MovieListPage/MovieListPage.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import SearchForm from '../SearchForm/SearchForm';
+import MovieDetails from '../MovieDetails/MovieDetails';
+import GenreSelect from '../GenreSelect/GenreSelect';
+import SortControl from '../SortControl/SortControl';
+import MovieTile from '../MovieTile/MovieTile';
+import './movie-list.scss';
+
+const MovieListPage = () => {
+    const [searchQuery, setSearchQuery] = useState('');
+    const [sortCriterion, setSortCriterion] = useState('Release Date');
+    const [activeGenre, setActiveGenre] = useState('all');
+    const [movieList, setMovieList] = useState([]);
+    const [selectedMovie, setSelectedMovie] = useState(null);
+    const [totalMovies, setTotalMovies] = useState(0);
+    const [searchDone, setSearchDone] = useState(false);
+
+    const genres = ['all', 'documentary', 'comedy', 'horror', 'crime'];
+
+    useEffect(() => {
+        async function fetchMyAPI() {
```

Comment:
My recommendation is don't store fetch logic inside component, better to create separate api file for such cases